.buildbot.sh

```diff
@@ -131,11 +131,15 @@ export YKB_YKLLVM_BUILD_ARGS="define:CMAKE_C_COMPILER=/usr/bin/clang,define:CMAK
 cargo -Z unstable-options build --release --build-plan -p ykcapi | \
     awk '/yk_testing/ { ec=1 } /yk_jitstate_debug/ { ec=1 } END {exit ec}'
 
-cargo build --release -p ykcapi
-
-for i in $(seq 10); do
-    RUST_TEST_SHUFFLE=1 cargo test --release
-    YKD_NEW_CODEGEN=1 RUST_TEST_SHUFFLE=1 cargo test --release
+tracers="hwt swt"
+
+for tracer in $tracers; do 
+    YKB_TRACER="${tracer}" cargo build --release -p ykcapi
+    echo "Running ${tracer} tests"
+    for i in $(seq 10); do
+        YKB_TRACER="${tracer}" RUST_TEST_SHUFFLE=1 cargo test --release
```

Comment:
Just realised: we should also test this in non-release mode (which should be a bit further up in the `.buildbot.sh` file).

Reply:
Updated 👉 9c5c476120b2b886d4a71cffa3f1760916f330b3